amundsen_application/static/js/config/config-utils.ts

```diff
@@ -30,3 +30,17 @@ export function getDatabaseIconClass(databaseId: string): string {
 
   return databaseConfig.iconClass;
 }
+
+/**
+ * Returns whether or not feedback features should be enabled
+ */
+export function feedbackEnabled(): boolean {
+  return AppConfig.mailClientFeatures.feedbackEnabled;
+}
+
+/**
+ * Returns whether or not notification features should be enabled
+ */
+export function notificationsEnabled(): boolean {
+  return AppConfig.mailClientFeatures.notificationsEnabled;
```

Comment:
Just something to ponder:

As an alternative to exporting a function to check each feature, I wonder if we could make a generic function that says `checkFeature('notifications')` or something. The downside being that you'd need to make `'notifications'` a constant and export that as well.

Reply:
Yeah i think a better way will expose itself soon enough. But I'm unsure of what's best at this point given that our configs vary so much. A good majority of the helper methods will just return the config value itself, while others like database display name and icon class methods execute some extra logic as well to handle defaults. 

But yes ideally it would be great to just know to import a helper function that takes parameters as opposed to importing various ones.